### PR TITLE
Make create_color func more resistant to non-digit characters

### PR DESCRIPTION
**Pull request by HappyCthulhu**
Now if this func took non-digital symbols, it return [0, 0, 0] in all cases

### CODE REVIEW

File: tests/test_sisulib.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    def test_with_letters(self):
+        assert [0, 0, 0] == create_color('128, 12a, 128')
+
+    def test_with_less_len_list(self):
+        assert [0, 0, 0] == create_color('135, 125')
+
+
+class TestGetRelatedLayersFunc:
+    def test_get_derived_only(self):
+        related_layers = get_related_layers(read_sisufile(PATH_TO_CONFIG_FILE))
```
Comment (tmshv) [inline tests/test_sisulib.py:18]: test `read_sisufile` in separate test case.
put a constant here or open file directly

**Review by tmshv — CHANGES_REQUESTED** [review]

File: tests/test_sisulib.py (hunk 1 above)
Comment (HappyCthulhu) [reply tests/test_sisulib.py]: replaced with constant

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+packaging==20.9
+pathlib2==2.3.5
+pluggy==0.13.1
+py==1.10.0
+pyparsing==2.4.7
+pytest==4.6.11
+scandir==1.10.0
+six==1.15.0
+wcwidth==0.2.5
+zipp==1.2.0
```
Comment (tmshv) [inline requirements.txt:18]: what the hell is this? just `pytest` isnt enough?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+FROM python:2.7.18-alpine3.11
+
+WORKDIR ./app
+
+COPY . .
+
+RUN pip install -r requirements.txt
+
+CMD ["pytest", "tests"]
+
```
Comment (tmshv) [inline Dockerfile:10]: please

remove

this

empty

lines

but

keep

EOL

File: .dockerignore [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# TODO: optimize this
```
Comment (tmshv) [inline .dockerignore:1]: what you wanted to optimize?

File: .dockerignore [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+tests/__pycache__
+__plugin__.py
```
Comment (tmshv) [inline .dockerignore:3]: we need this file inside a container

**Review by tmshv — CHANGES_REQUESTED** [review]

File: .dockerignore (hunk 4 above)
Comment (HappyCthulhu) [reply .dockerignore:1]: maybe its possible to ingore __pycache__ with one, not two strings

File: requirements.txt (hunk 2 above)
Comment (HappyCthulhu) [reply requirements.txt:18]: its pip freeze stuff

Comment (tmshv) [reply requirements.txt]: almost all of them is transitive dependencies
add only direct ones

File: .dockerignore (hunk 4 above)
Comment (tmshv) [reply .dockerignore:1]: yep just keep `__pycache__` line

File: requirements.txt (hunk 2 above)
Comment (HappyCthulhu) [reply requirements.txt]: corrected

File: Dockerfile (hunk 3 above)
Comment (HappyCthulhu) [reply Dockerfile:6]: fixed

File: .dockerignore (hunk 5 above)
Comment (HappyCthulhu) [reply .dockerignore]: fixed

File: tests/test_sisulib.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import os
+import sys
+
+sys.path.append(os.path.abspath('./tests/'))
+from sisulib import create_color, get_related_layers, read_sisufile
+
+PATH_TO_CONFIG_FILE = 'sisufile.json'
```
Comment (tmshv) [inline tests/test_sisulib.py:7]: unused variable

File: tests/test_sisulib.py [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+        assert [0, 0, 0] == create_color('128, 12a, 128')
+
+    def test_with_less_len_list(self):
+        assert [0, 0, 0] == create_color('135, 125')
+
+
+class TestGetRelatedLayersFunc:
+    def test_get_derived_only(self):
+        json = {u'version': u'0', u'data': [
+            {u'layer': [u'Default', {u'color': [255, 0, 0], u'lineWeight': 1, u'lineType': u'continuous'}],
```
Comment (tmshv) [inline tests/test_sisulib.py:24]: why `u''`?

File: tests/test_sisulib.py [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    def test_with_letters(self):
+        assert [0, 0, 0] == create_color('128, 12a, 128')
+
+    def test_with_less_len_list(self):
+        assert [0, 0, 0] == create_color('135, 125')
+
+
+class TestGetRelatedLayersFunc:
+    def test_get_derived_only(self):
+        json = {u'version': u'0', u'data': [
```
Comment (tmshv) [inline tests/test_sisulib.py:23]: variable is not a json
it is a config

File: tests/test_sisulib.py [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+            {u'layer': [u'Default', {u'color': [255, 0, 0], u'lineWeight': 1, u'lineType': u'continuous'}],
+             u'code': u'X', u'options': {}, u'properties': {u'patternBasePoint': [0, 0, 0], u'patternRotation': 0},
+             u'view': [{u'render': [u'hatch',
+                                    {u'color': [10, 190, 10], u'pattern': u'Solid', u'scale': 1, u'lineWeight': 0.13}],
+                        u'layerSuffix': u'_SOLID'}, {u'render': [u'hatch', {u'color': [0, 255, 0], u'pattern': u'Grid',
+                                                                            u'scale': 1, u'lineWeight': 0.1}],
+                                                     u'layerSuffix': u'_HATCH'}]}]}
```
Comment (tmshv) [inline tests/test_sisulib.py:30]: plz format this shit
i can't read it

**Review by tmshv — CHANGES_REQUESTED** [review]
and add a Test section to readme with docker commands

```
docker build -t designunit/sisu .
docker run --rm --name sisu designunit/sisu
```

**Review by tmshv — APPROVED** [review]

**Pull request merged**